app/lib/.server/llm/stream-text.ts

```diff
@@ -88,7 +90,10 @@ export async function streamText(props: {
 
   const systemPrompt = getAgent8Prompt(WORK_DIR);
 
-  let combinedTools: Record<string, any> = { ...tools };
+  const docTools = await createDocTools(serverEnv as Env);
+  const codebaseTools = await createSearchCodebase(serverEnv as Env);
+  const resourcesTools = await createSearchResources(serverEnv as Env);
+  let combinedTools: Record<string, any> = { ...tools, ...docTools, ...codebaseTools, ...resourcesTools };
```

Comment:
`stream-text` also seems to be used by `api.llmcall.ts` which just selects template. are these tools needed those usages too?

Reply:
Right, so I am also making another logic for llmcall.